Analytical_Engine/Query/Graph/IsolatedEntities.cs

```diff
@@ -43,6 +43,12 @@ public static partial class Query
         [Output("isolated entities", "The collection of entity Guids that are isolated.")]
         public static List<Guid> IsolatedEntities(this Graph graph)
         {
+            if(graph == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the isolated entities of a null graph.");
+                return null;
```

Comment:
Why not return an empty collection to follow the pattern?
```suggestion
                return new List<Guid>();
```